examples_tests/31.SkinningDataBenchmark/3.vert

```diff
@@ -0,0 +1,34 @@
+#version 460 core
+
+layout( push_constant, row_major ) uniform Block 
+{
+    mat4 vp;
+    uvec4 modelMatrixOffset;
+    uvec4 normalMatrixOffset;
+	
+}pc;
+
+layout(std430, set = 0, binding = 0, row_major) restrict readonly buffer BoneMatrices
+{
+    vec3 column[];

```

Comment:
ROWS store ROWS not columns

A `vec3` array is basically a `vec4` array but with useless padding.